src/main/scala/rocketchip/Configs.scala

```diff
@@ -28,8 +28,9 @@ class BasePlatformConfig extends Config(
     case PeripheryBusArithmetic => true
     // Note that PLIC asserts that this is > 0.
     case IncludeJtagDTM => false
-    case ExtMem => AXIMasterConfig(0x80000000L, 0x10000000L, 8, 4)
-    case ExtBus => AXIMasterConfig(0x60000000L, 0x20000000L, 8, 4)
+    case ExtMem => MasterConfig(0x80000000L, 0x10000000L, 8, 4)
+    case ExtBus => MasterConfig(0x60000000L, 0x20000000L, 8, 4)
+    case ExtIn  => SlaveConfig(8, 8, 2)
     case RTCPeriod => 100 // gives 10 MHz RTC assuming 1 GHz uncore clock
```

Comment:
I've made the changes you requests. If you are happy with them, please approve the PR so I can merge it.